storey/flow.py

```diff
@@ -1191,15 +1196,199 @@ async def _load_schema(self, container, table_path):
+            else:
+                raise V3ioError(
+                    f'Failed to save aggregation for {table_path}/{key}. Response status code was {response.status_code}: {response.body}')
+        else:
+            raise V3ioError(
+                f'Failed to save aggregation for {table_path}/{key}. Response status code was {response.status_code}: {response.body}')
+
+    async def _fetch_state_by_key(self, aggr_item, container, table_path, key):
+        attributes_to_get = self._get_time_attributes_from_aggregations(aggr_item)
+        get_item_response = await self._v3io_client.kv.get(container, table_path, key, attribute_names=attributes_to_get,
+                                                           raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+        if get_item_response.status_code == 200:
+            aggr_item.storage_specific_cache[self._mtime_header_name] = get_item_response.headers[self._mtime_header_name]
+
+            # First reset all relevant cache items
+            for bucket in aggr_item.aggregation_buckets.values():
+                if bucket.should_persist:
+                    for attribute_to_reset in attributes_to_get:
+                        if attribute_to_reset.startswith(f'{self._aggregation_time_attribute_prefix}{bucket.name}_'):
+                            bucket.storage_specific_cache.pop(attribute_to_reset, None)
+
+            for name, value in get_item_response.output.item.items():
+                for bucket in aggr_item.aggregation_buckets.values():
+                    if bucket.should_persist:
+                        if name.startswith(f'{self._aggregation_time_attribute_prefix}{bucket.name}_'):
+                            bucket.storage_specific_cache[name] = int(value.timestamp() * 1000)
+        else:
+            raise V3ioError(
+                f'Failed to query {table_path}/{key}. Response status code was {get_item_response.status_code}: {get_item_response.body}')
+
+    def _get_time_attributes_from_aggregations(self, aggregation_element):
+        attributes = {}
+        for bucket in aggregation_element.aggregation_buckets.values():
+            attributes[f'{bucket.name}_a'] = f"{self._aggregation_time_attribute_prefix}{bucket.name}_a"
+            attributes[f'{bucket.name}_b'] = f"{self._aggregation_time_attribute_prefix}{bucket.name}_b"
+        return list(attributes.values())
+
+    def _is_false_condition_error(self, response):
+        if response.status_code == 400:
+            body = str(response.body)
+            if self._false_condition_inner_error_code in body and self._false_condition_outer_error_code in body and body.count(
+                    self._error_code_string) == 2:
+                return True
+        return False
+
+    def _build_feature_store_update_expression(self, aggregation_element, additional_data, partitioned_by_key, pending=None):
+        condition_expression = ""
```

Comment:
Should be None, or this will get passed to v3io-py and it will actually send `"ConditionExpression": ""` as part of the request. Missing a test?